src/GridFS/StreamWrapper.php

```diff
@@ -88,6 +96,10 @@ public static function register($protocol = 'gridfs')
      */
     public function stream_close()
     {
+        if (! $this->stream) {
```

Comment:
Is `stream_close` the only handler that might be called after StreamWrapper destructs? Just want to make sure we don't need `if (! $this->stream)` checks elsewhere.

Reply:
PHP only calls `stream_close` after calling `__destruct`. Userland code shouldn't manage to make such a call in the first place.